webapp/packages/plugin-navigation-tree/src/TreeNew/Tree.tsx

```diff
@@ -27,7 +28,7 @@ import { useTreeVirtualization } from './useTreeVirtualization.js';
 import { TreeMenuContextProvider } from './contexts/TreeMenuContext/TreeMenuContextProvider.js';
 import type { ITreeMenu } from './useTreeMenu.js';
 
-export interface NavigationTreeNewProps {
+export interface NavigationTreeNewProps extends React.PropsWithChildren {
```

Comment:
Or we can use `PropsWithChildren<NavigationTreeNewProps> `(optional but I think its more suitable than extend)